CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 1.0
+## 1.1.0 – [diff](https://github.com/openfisca/openfisca-france/compare/1.0...1.1.0)
```

Comment:
I doesn't matter much, but just for our SemVer reflexion, I wonder what kind of version bump was the most appropriate here.

On the one hand, it's mostly internal business: the behaviour of the variables will stay the same, they are just declared in another way. That would be an argument towards a patch release.

On the other hand, if a contributor has written a reform using a `build_column`, his.her code would be broken now. That could even push for a major release...

I guess the main problem is the fact that `base.py` explicitly list the functions to import from core. It may be something to clean up to improve dependency management.